### DISCUSSION TITLE
guide/drivers/pikpak

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#355] [Announcements]
# guide/drivers/pikpak

PikPak Username email or phone? Password password Root folder id Can get with https://mypikpak.com/ , default root. image.png Disable media link https://github.com/alist-org/ali...

https://alist.nn.ci/guide/drivers/pikpak.html

**Comment by ohnggni** [comment]
After recent update(3.35.0), the setting of Pikpak gets to have 'client id' and 'client secret'. What values should I put in? Now every morning, Pikpak in AList is having connection issue. Once I remove those values(entered in default), then it works again. However, when I check the setting again, arbitrary values are still there. How can I resolve this problem?

**Reply by ohnggni** [reply]
<img width="1324" alt="스크린샷 2024-06-05 09 05 41" src="https://github.com/alist-org/docs/assets/33609687/ef1bd86d-67ed-4f95-b82e-d475cfee99bf">
Please see this screen-shot. I don't know where those come from.

**Comment by rsndm1413** [comment]
Failed init storage: Post "https://user.mypikpak.com/v1/auth/signin": read tcp ***********:5351->***********83:443: wsarecv: An existing connection was forcibly closed by the remote host. How can I resolve this problem?
